### PR TITLE
add odin support

### PR DESCRIPTION
**Pull request by spsandwichman**
#490 
Adds support for [Odin](https://odin-lang.org/)

original logo:
emblem-260

adapted to ascii:
```
                 @@@@
         @@@@@@@@@@@@    @@@@
      @@@@@@    @@@@    @@@@@@@@
   @@@@@      @@@@    @@@@    @@@@
  @@@@       @@@@    @@@@       @@@@
 @@@       @@@@@   @@@@          @@@@
@@@       @@@@    @@@@            @@@@
@@@      @@@@   @@@@               @@@
@@@    @@@@    @@@@                @@@
@@@   @@@@   @@@@                  @@@
@@@@@@@@    @@@@                  @@@@
 @@@@@@   @@@@@                  @@@@
   @@    @@@@                  @@@@
        @@@@                @@@@@
       @@@@@@@@@      @@@@@@@@
            @@@@@@@@@@@@@@@
```

preview:
output

### CODE REVIEW

**Review by spenserblack — APPROVED** [review]
Thanks!

Approving, but you might want to consider styling this a bit. For example, since the original logo's background has a gradient, you might want to use true color (hex) to add a gradient to the ascii art.

**Comment by spsandwichman** [comment]
awesome, I'll experiment with it when I can!

**Comment by spsandwichman** [comment]
updated with gradient
image

File: languages.yaml [hunk 1]
```diff
@@ -1639,6 +1639,36 @@ OCaml:
...
+    ansi:
+      - blue
+      - blue
+      - blue
+    hex:
+      - "#64B9FA"
+      - "#189CFF"
+      - "#107BCC"
+    chip: "#60AFFE"
+
```
Comment (o2sh) [inline languages.yaml:1671]: remove empty line

File: languages.yaml [hunk 2]
```diff
@@ -1639,6 +1639,36 @@ OCaml:
...
+  colors:
+    ansi:
+      - blue
+      - blue
+      - blue
+    hex:
+      - "#64B9FA"
+      - "#189CFF"
+      - "#107BCC"
+    chip: "#60AFFE"
```
Comment (o2sh) [inline languages.yaml:1670]: Great first design @spsandwichman, I took some time to clean up your ASCII art to make it more symmetrical and I also added a background:

image


I'd suggest you follow @spenserblack's suggestion and apply the gradient to the background


```
Odin:
  type: programming
  ascii: |
    {0}@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
    {0}@@@@@@@@@@@@@@@{1}@@@@@@@@@{0}@@@@@@@@@@@@@@@@
    {0}@@@@@@@@@@@{1}@@@@{0}@@@@@{1}@@@{0}@@@{1}@@@@{0}@@@@@@@@@@
    {0}@@@@@@@@{1}@@@{0}@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@{1}@@@{0}@@@@@@@@
    {0}@@@@@@{1}@@@{0}@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@{1}@@@{0}@@@@@@
    {0}@@@@{1}@@@{0}@@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@{1}@@@{0}@@@@
    {0}@@@{1}@@@{0}@@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@{1}@@@{0}@@@
    {0}@@{1}@@@{0}@@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@@{1}@@@{0}@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@@{1}@@@{0}@@@
    {0}@@@@{1}@@@{0}@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@@{1}@@@{0}@@@@
    {0}@@@@@@{1}@@@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@{1}@@@{0}@@@@@@
    {0}@@@@@@@@{1}@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@{1}@@@{0}@@@@@@@@
    {0}@@@@@@@@@@@@{1}@@@{0}@@@@@@@@@@{1}@@@@{0}@@@@@@@@@@@
    {0}@@@@@@@@@@@@@@@{1}@@@@@@@@@@{0}@@@@@@@@@@@@@@@
    {0}@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
  colors:
    ansi:
      - blue
      - white
    hex:
      - "#3882D2"
      - "#FFFFFF"
    chip: "#60AFFE"
```

Comment (o2sh) [reply languages.yaml:1670]: Here is a second revision:

image

Thinking about it, incorporating the gradient to the background might be challenging, It's up to you @spsandwichman if you want to try.


```
Odin:
  type: programming
  ascii: |
    {0}oooooooooooooooooooooooooooooooooooo
    {0}oooooooooooooooooooooooooooooooooooo
    {0}ooooooooooooo{1}@@@@@@@@@{0}oooooooooooooo
    {0}ooooooooo{1}@@@@{0}ooooo{1}@@@{0}ooo{1}@@@{0}ooooooooo
    {0}ooooooo{1}@@@@{0}oooooo{1}@@@{0}ooo{1}@@@@@@{0}ooooooo
    {0}ooooo{1}@@@@{0}ooooooo{1}@@@{0}ooo{1}@@@{0}oo{1}@@@@{0}ooooo
    {0}oooo{1}@@@{0}oooooooo{1}@@@{0}ooo{1}@@@{0}ooooo{1}@@@{0}oooo
    {0}ooo{1}@@@{0}oooooooo{1}@@@{0}ooo{1}@@@{0}ooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}ooooooo{1}@@@{0}ooo{1}@@@{0}oooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}oooooo{1}@@@{0}ooo{1}@@@{0}ooooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}ooooo{1}@@@{0}ooo{1}@@@{0}oooooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}oooo{1}@@@{0}ooo{1}@@@{0}ooooooooooo{1}@@@{0}ooo
    {0}oooo{1}@@@{0}oo{1}@@@{0}ooo{1}@@@{0}ooooooooooo{1}@@@{0}oooo
    {0}ooooo{1}@@@@@@{0}ooo{1}@@@{0}oooooooooo{1}@@@@{0}ooooo
    {0}ooooooo{1}@@@{0}ooo{1}@@@{0}ooooooooo{1}@@@@{0}ooooooo
    {0}oooooooooooo{1}@@@{0}oooooooo{1}@@@@{0}ooooooooo
    {0}ooooooooooooo{1}@@@@@@@@@@{0}ooooooooooooo
    {0}oooooooooooooooooooooooooooooooooooo
    {0}oooooooooooooooooooooooooooooooooooo
  colors:
    ansi:
      - blue
      - white
    hex:
      - "#3882D2"
      - "#FFFFFF"
    chip: "#60AFFE"
```

Comment (spsandwichman) [reply languages.yaml:1670]: would you mind if i removed the background and implemented a blue gradient with the O itself? I personally like the no-background look, but it's totally your call, and thank you!

Comment (o2sh) [reply languages.yaml:1670]: Of course go ahead 👍  Take the second revision as a starting to point for that and remove the BG and apply the gradient to the logo itself

**Pull request merged**